tools/scripts/BuildZippedCLI.ps1

```diff
@@ -66,22 +66,47 @@ function Create-Zipfile([string]$srcDir, [string]$app, [string]$zipFile, [string
     $scratch=(Get-UniqueTempFolder $app)[0]
     Write-Verbose "scratch = $scratch"
 
-    Copy-Files $srcDir '*.*' $scratch
+    Prepare-Zipfile $srcDir $otherSignedSrcDir $scratch $patternsToRemove $otherSignedAssemblies
```

Comment:
Umm... does this run? PowerShell requires functions to appear _before_ the caller. I'd expect an error from calling Prepare-ZipFile before it's defined

Reply:
Yes, you can find its output in the artifacts of the signed build linked to above.

Reply:
Got it--Create-ZipFile is called from CreateCLIZip, which appears _after_ Prepare-ZipFile. By the time the interpreter executes the call on line 69, it's already encountered Prepare-ZipFile, so it works.

Just to be safe for future changes, you might consider moving Prepare-ZipFile so it appears before Create-ZipFile

Reply:
Order updated